packages/botonic-nlp/src/dataset/loader.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { lstatSync, readdirSync, readFileSync } from 'fs'
+import * as yaml from 'js-yaml'
```

Comment:
you do not need to require all the objects, but just the ones you are using:
`import {load as loadYaml} from 'js-yaml'`